src/types/basic/uint.ts

```diff
@@ -1,15 +1,23 @@
 import {Json} from "../../interface";
-import {BasicType} from "./abstract";
+import {BasicType, isTypeOf} from "./abstract";
 
 export interface IUintOptions {
   byteLength: number;
 }
 
+export const UINT_TYPE = Symbol.for("ssz/UintType");
+
+export function isUintType<T>(type: unknown): type is UintType<T> {
+  return isTypeOf(type, UINT_TYPE);
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
are you planning on working towards removing these eslint-disable's?

Reply:
whoops, it looks like a few are superfluous